<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilesGeneratorDockWidgetBase</class>
 <widget class="QDockWidget" name="TilesGeneratorDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generator Kafli</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="cportal_btn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources.qrc">
        <normaloff>:/plugins/tiles_generator/c-geoportal.png</normaloff>:/plugins/tiles_generator/c-geoportal.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="path_input">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>ścieżka zapisu kafli</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show_path_btn">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="zip_check">
      <property name="text">
       <string>Spakuj do ZIP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="layers_group">
      <property name="title">
       <string>Warstwy</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="all_layers_btn">
         <property name="text">
          <string>Wszystkie warstwy</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="visible_layers_btn">
         <property name="text">
          <string>Warstwy widoczne na mapie</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="zoom_group">
      <property name="title">
       <string>Poziomy powiększenia</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QSlider" name="max_slider">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="min_slider">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="max_display_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="min_display_label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="min_label">
         <property name="text">
          <string>Minimum</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="max_label">
         <property name="text">
          <string>Maksimum</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="range_group">
      <property name="title">
       <string>Zasięg</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="layers_range_btn">
         <property name="text">
          <string>Zasięg warstw mapy</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="map_range_btn">
         <property name="text">
          <string>Widoczny zasięg mapy</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="user_range_btn">
           <property name="text">
            <string>Zasieg użytkownika</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="draw_extent_btn">
           <property name="text">
            <string>Rysuj</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="styleSheet">
         <string notr="true">QProgressBar{text-align: center;
                       border: 1px solid rgb(48, 50, 55);
				       border-radius: 10px;
                       background-color: rgb(0, 0, 0, 0)}
QProgressBar::chunk{background-color: rgb(5, 151, 49, 125);
                                    border-radius: 6px;}</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="progress_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generate_tiles_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Generuj kafle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources/resources.qrc">
          <normaloff>:/plugins/tiles_generator/reject.png</normaloff>:/plugins/tiles_generator/reject.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
